v3/src/components/graph/adornments/univariate-measures/median/median-adornment-model.ts

```diff
@@ -0,0 +1,25 @@
+import { Instance } from "mobx-state-tree"
+import { median } from "mathjs"
+import {
+  UnivariateMeasureAdornmentModel, IUnivariateMeasureAdornmentModel
+  } from "../univariate-measure-adornment-model"
+import { kMedianType } from "./median-adornment-types"
+import { IDataConfigurationModel } from "../../../../data-display/models/data-configuration-model"
+
+export const MedianAdornmentModel = UnivariateMeasureAdornmentModel
+  .named("MedianAdornmentModel")
+  .props({
+    type: "Median"
```

Comment:
I should have pointed this out earlier, but this should be
```suggestion
    type: types.optional(types.literal(kMedianType), kMedianType)
```
for all adornments (and any other models where types are used to discriminate between models). As written, the TypeScript type is `string` with a default value of `"Median"`, whereas `types.literal()` indicates that there is only one valid value, and `types.optional()` means that it will be provided automatically if not provided by the caller. Once all adornments are using `types.literal()` we should test whether the explicit dispatcher is still necessary.